### PR TITLE
feat(repo): add jenkins-node to CI

### PR DESCRIPTION
**Pull request by bacongobbler**
See https://github.com/deis/jenkins-node

### CODE REVIEW

File: repo.groovy [hunk 1]
```diff
@@ -30,6 +30,11 @@ repos = [
     workflowComponent: true,
     chart: 'fluentd'],
 
+  [ name: 'jenkins-node',
```
Comment (vdice) [inline repo.groovy:33]: @bacongobbler do we also want the usual component build and release jobs to be auto-generated for `jenkins-node` (really just necessitates that the project have a `Makefile` w/ `docker-build` and `docker-push` targets)?  

If no, we can add an add'l key:value of `buildJobs: false` in this same entry -- then the only job that will be generated will be `jenkins-node-chart-publish` -- which is currently usually kicked off as part of a component's release pipeline (which we wouldn't have w/ `buildJobs: false` mentioned above), but can be kicked off manually as well.

Comment (bacongobbler) [reply repo.groovy:33]: Certainly! There's no Makefile in the project at the moment but I can cook one up in a sec. :)

Comment (bacongobbler) [reply repo.groovy:33]: Done. Is there anything else that I need to do here?

Comment (vdice) [reply repo.groovy:33]: @bacongobbler Nope, _should_ be g2g!

**Pull request merged**